<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>pqHandleWidget</class>
 <widget class="QWidget" name="pqHandleWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="show3DWidget" >
       <property name="text" >
        <string>Visible</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="useCenterBounds" >
       <property name="text" >
        <string>Center on Bounds</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="worldPositionX" />
     </item>
     <item>
      <widget class="QLineEdit" name="worldPositionY" />
     </item>
     <item>
      <widget class="QLineEdit" name="worldPositionZ" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>show3DWidget</tabstop>
  <tabstop>useCenterBounds</tabstop>
  <tabstop>worldPositionX</tabstop>
  <tabstop>worldPositionY</tabstop>
  <tabstop>worldPositionZ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
